business-registry-dashboard/app/stores/affiliations.ts

```diff
@@ -69,6 +72,16 @@ export const useAffiliationsStore = defineStore('brd-affiliations-store', () =>
     }
   }
 
+  /* Check if Business can be deleted safely (i.e. does not have a review record) */
+  function canBusinessBeDeleted (payload: RemoveBusinessPayload) {
+    // For now only including Continuation Ins where only draft records can be deleted
+    if (payload.business.corpType.code === CorpTypes.CONTINUATION_IN && payload.business?.draftStatus !== EntityStates.DRAFT) {
```

Comment:
Note that the second condition will automatically be True if business is falsy (ie, undefined !== DRAFT). Ditto with draftStatus.

I get around this by first ensuring `payload.business?.draftStatus` is truthy.